test/unit/application_keys_test.rb

```diff
@@ -159,11 +158,10 @@ def test_format
   def test_regenerate_key
     @application_keys.add(key = 'app-key')
     @application.reload
+    Timecop.travel(2.hours.from_now)
     updated_at = @application.updated_at
     @application_keys.regenerate(key)
-    # @application.reload
```

Comment:
Good removing this, we do not need it because it is the same instance so it is reloaded